bcda/bcdacli/cli.go

```diff
@@ -158,7 +163,7 @@ func setUpApp() *cli.App {
 				},
 				cli.StringFlag{
 					Name:        "group-id",
-					Usage:       "Auth group ID (required for SSAS)",
+					Usage:       "(Optional) Auth group ID",
```

Comment:
I like the flexibility in making `group-id` optional for `create-aco` and `aco-id` optional for `create-group`.  It allows us to execute the commands in any order we choose.
However, do we need any error handling for a scenario where we call `create-aco` with no `group-id` and then call `create-group` with no `aco-id`?  What would happen if an engineer ran the commands like that?

Reply:
The ACO and group would be created but not associated with each other. Relates to my question of if we need a command to tie existing groups and ACOs together. Do you have an idea for how we'd identify that there's an error? Should we require a certain order instead?